localnet/kubernetes/values-relayminer-1.yaml

```diff
@@ -4,22 +4,4 @@ config:
   signing_key_name: supplier1
 
   default_signing_key_names: [supplier1]
-  suppliers:
-    - service_id: anvil
-      listen_url: http://0.0.0.0:8545
-      service_config:
-        backend_url: http://anvil:8547/
-        publicly_exposed_endpoints:
-          - relayminer1
-    - service_id: ollama
-      listen_url: http://0.0.0.0:8545
-      service_config:
-        backend_url: http://ollama:11434/
-        publicly_exposed_endpoints:
-          - relayminer1
-    - service_id: rest
-      listen_url: http://0.0.0.0:8545
-      service_config:
-        backend_url: http://rest:10000/
-        publicly_exposed_endpoints:
-          - relayminer1
+  suppliers: [] # suppliers list is dynamically defined in poktroll/Tiltfile.
```

Comment:
I just want a 👍 from @okdas in case this has downstream effects on our E2E testing in ephemeral DevNets.

Reply:
Thanks for the feedback @okdas